source/_integrations/zwave_js.markdown

```diff
@@ -98,13 +98,14 @@ While your Z-Wave mesh is permanently stored on your dongle, the additional meta
 
 ### Adding a new device to the Z-Wave network
 
-1. In Home Assistant, go to <a class="my" href="https://my.home-assistant.io/redirect/integrations/" target="_blank"><b>Settings</b> > <b>Devices & Services</b></a>.
+1. In Home Assistant, go to {% my integrations title="**Settings** > **Devices & Services**" %}.
 1. In the Z-Wave integration, select **Configure**.
 1. Select **Add device**.
    * The Z-Wave controller is now in inclusion mode.
-1. Put the device you want to add in inclusion mode. Refer to the device manual to see how this is done.
-   * The UI should confirm that the node was added. The device will be immediately visible in Home Assistant. After a short while (seconds to minutes) the entities should also be created.
+1. If your device supports SmartStart, select **Scan QR code** and scan the QR code on your device.
```

Comment:
Note that scanning a QR code requires https in the frontend that is used to do this. So it will not be available for all users even if the device supports SmartStart. Not sure how much detail we want to explain here.

Reply:
I think the UI is pretty helpful here. I did not have https when  did it and the UI helped me through it. I managed to add it without any issues. (And I am not a very technical user). I would not document every detail here.

Reply:
Sounds good.